oelint_adv/__main__.py

```diff
@@ -37,11 +37,11 @@
 class TypeSafeAppendAction(argparse.Action):
 
     def __call__(self, parser, namespace, values, option_string=None):
+        if not isinstance(values, str):
+            return  # pragma: no cover
         items = getattr(namespace, self.dest) or []
-        if isinstance(items, str):
-            items = RegexRpl.split(r'\s+|\t+|\n+', items)  # pragma: no cover
-        items.append(values)  # pragma: no cover
-        setattr(namespace, self.dest, items)  # pragma: no cover
+        items += RegexRpl.split(r'\s+|\t+|\n+', values.strip('"').strip("'"))
```

Comment:
+= is slower then .extend and discouraged because it creates a new copy of the initial list.

Reply:
thanks, today I learned something :-) - fixed